### PR TITLE
Latest-fixes-in-release-050

### PR DESCRIPTION
**Pull request by y0014984**

### CODE REVIEW

File: addons/armaos/functions/fnc_shell_findLoginUser.sqf [hunk 1]
```diff
@@ -17,16 +17,25 @@ private _users = _computer getVariable ["AE3_Userlist", createHashMap];
 
 private _result = [];
 
-if (_username in _users) then 
+if (_username == "root") then
```
Comment (GermanHydrogen) [inline addons/armaos/functions/fnc_shell_findLoginUser.sqf:20]: Permits root access in general. 
To fix:
```
if (_username in _users) then
{...}else
{
   if (_username isEqualTo "root") then
   {...};
};
```

File: addons/armaos/XEH_preInit.sqf [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+		["US", "DE"],
+		[["US", "United States"], ["DE", "Deutschland"]], 
+		0
+	],
+    nil, // "_isGlobal" flag. Set this to true to always have this setting synchronized between all clients in multiplayer
+    {  
+        params ["_value"];
+    }, // function that will be executed once on mission start and every time the setting is changed.
+    true // Setting will be marked as needing mission restart after being changed. (optional, default false) <BOOL>
+] call CBA_fnc_addSetting;
```
Comment (GermanHydrogen) [inline addons/armaos/XEH_preInit.sqf:19]: This is set per mission/server right?

**Review by GermanHydrogen — CHANGES_REQUESTED** [review]

File: addons/armaos/XEH_preInit.sqf (hunk 2 above)
Comment (y0014984) [reply addons/armaos/XEH_preInit.sqf:19]: All CBA settings are client-side by default as long as `_isglobal == false`, but mission makers and server admins can decide to overwrite client-settings with their mission or server settings, see [here](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System).

File: addons/armaos/functions/fnc_shell_findLoginUser.sqf (hunk 1 above)
Comment (y0014984) [reply addons/armaos/functions/fnc_shell_findLoginUser.sqf:20]: I will change that.

**Review by GermanHydrogen — APPROVED** [review]

**Pull request merged**